src/installer-source/KolibriSetupScript.iss

```diff
@@ -600,7 +600,14 @@ begin
             'System\CurrentControlSet\Control\Session Manager\Environment',
             'KOLIBRI_SETUP'
         )
-        Exec('cmd.exe', '/c "reg delete HKCU\Environment /F /V KOLIBRI_SETUP"', '', SW_HIDE, ewWaitUntilTerminated, ErrorCode)   
+        Exec('cmd.exe', '/c "reg delete HKCU\Environment /F /V KOLIBRI_SETUP"', '', SW_HIDE, ewWaitUntilTerminated, ErrorCode)
+        {Set kolibri command to Path}
+        RegWriteStringValue(
+            HKLM,
+            'System\CurrentControlSet\Control\Session Manager\Environment',
+            'Path',
+            GetPipDir('')
+        );   
```

Comment:
Does this overwrite the existing `Path` env or append to it?

Reply:
> Does this overwrite the existing Path env or append to it?

I tested this and the Kolibri env will just append to Path. I'm still doing testing on other scenarios and added a Work in progress to this. Thanks for your comment @benjaoming

Reply:
@mrpau-richard Sorry that you had to wait for the testing. 
It overwrites the system `Path` variable. I was lucky to have opened a `cmd` window prior to the testing in which the old `Path` could be echoed and restored. :)

Let's also clearly consider the need to append or prepend. I think we should append as existing system behaviour should not be impacted. But I am not going to insist on this unless we have a case reported against the existing design of prepending.

Yet again as I had suggested in #161, I maintain that we should append the `%KOLIBRI_SCRIPT_DIR%` instead of its value. I understand that the value may have been added to ensure the `Path`'s integrity at the time of uninstallation. I have created another issue for this - #166 .

This could be helpful - https://stackoverflow.com/a/46609047/2437751